molecule/redhat/molecule.yml

```diff
@@ -1,4 +1,11 @@
 ---
+# This molecule configuration file is suitable for testing Ansible
```

Comment:
Same question about `molecule-no-systemd.yml` and `molecule-with-systemd.yml` symlinks here.

Reply:
See [this comment](https://github.com/cisagov/ansible-role-xfce/pull/6#discussion_r540648406).